src/sinks/datadog_archives.rs

```diff
@@ -115,43 +140,112 @@ pub struct DatadogArchivesSinkConfig {
     acknowledgements: AcknowledgementsConfig,
 }
 
-#[derive(Deserialize, Serialize, Default, Debug, Clone)]
-#[serde(deny_unknown_fields)]
```

Comment:
Was this intentionally dropped?

Reply:
Actually, eh, I did intend to drop it but thinking through, this is still a valid use of it.

Reply:
Fixed in 18c22d4b8.